contracts/OriginalTokenBridgeUpgradable.sol

```diff
@@ -107,13 +119,20 @@ contract OriginalTokenBridgeUpgradable is TokenBridgeBaseUpgradable {
         require(to != address(0), "OriginalTokenBridge: invalid to");
         _checkAdapterParams(remoteChainId, PT_MINT, adapterParams);
 
+        uint withdrawalAmountLD = amountLD;
+        if (withdrawalFeeBps > 0) {
+            uint withdrawalFee = (amountLD * withdrawalFeeBps) / TOTAL_BPS;
```

Comment:
Please check this flow with max and min values for uint

Reply:
@leonprou why is that needed?

Reply:
The user sends a really big amount of tokens, when it gets multiplied by `withdrawalFeeBps` an integer overflow might happen.